Bond/iOS/Bond+UIView.swift

```diff
@@ -71,4 +72,18 @@ extension UIView {
       return d
     }
   }
+    
+  public var dynUserInteractionEnabled: Dynamic<Bool> {
+    if let d: AnyObject = objc_getAssociatedObject(self, &userInteractionEnabledUIView) {
+      return (d as? Dynamic<Bool>)!
+    } else {
+      let d = InternalDynamic<Bool>(self.userInteractionEnabled)
+      let bond = Bond<Bool>() { [weak self] v in if let s = self { s.userInteractionEnabled = v } }
```

Comment:
@ivanmoskalev Sorry for the delay in responding to this comment. The updated statement you recommend works fine however it is inconsistent with the implementation of other dynamic variables in the project. I do believe that all dynamic variables should be updated however that task is outside the scope of this branch.